docs/configuration/http_conn_man/rds.rst

```diff
@@ -28,7 +28,7 @@ route_config_name
   *(required, string)* The name of the route configuration. This name will be passed to the
   :ref:`RDS HTTP API <config_http_conn_man_rds_api>`. This allows an Envoy configuration with
   multiple HTTP listeners (and associated HTTP connection manager filters) to use different route
-  configurations.
+  configurations. The route_config_name must **not** contain ``:``.
```

Comment:
Yeah, on second thoughts, maybe best to avoid baking in these dependencies. I know that we've been discussing trying to relax the `:` constraint on clusters at some point (but not now), so if there is a cleaner way, e.g. using tuples and hashing on that, then maybe do that.